example/src/Demo.jsx

```diff
@@ -28,6 +28,7 @@ function Demo({ fetchSwagger, status, docs, oas, oauth }) {
           <ApiExplorer
             i18n={{
               locale: lang,
+              defaultLocale: lang,
```

Comment:
It will be better a static string as default locale (as en), what do you think?

Reply:
generally speaking I agree, I've no strong opinion for this example

```suggestion
              defaultLocale: 'en',
```